src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -768,6 +785,49 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 			{
 				AiNodeSetFlt( node, g_volumePaddingArnoldString, m_volumePadding );
 			}
+
+			if( m_volumeGrids && m_volumeGrids->readable().size() )
+			{
+				AtArray *array = ParameterAlgo::dataToArray( m_volumeGrids.get(), AI_TYPE_STRING );
+				AiNodeSetArray( node, g_volumeGridsArnoldString, array );
```

Comment:
This isn't checking that the node is a volume node or has a volume_grids parameter.